code/modules/mob/living/say.dm

```diff
@@ -243,6 +249,13 @@ var/list/department_radio_keys = list(
 				if(M.lingcheck() || (M in dead_mob_list && !istype(M, /mob/new_player)))
 					M << "<i><font color=#800080><b>[mind.changeling.changelingID]:</b> [message]</font></i>"
 			return 1
+	if((message_mode == MODE_HEADSET && construct_chat_check(0) /*sending check for constructs*/) || \

```

Comment:
I feel like this can be in /mob/living/simple_animal/construct/handle_inherent_channels() Just so humans and whatnot dont call construct_chat_check every time they talk.